client/src/index.jsx

```diff
@@ -18,37 +18,67 @@ const Heading2 = styled.h2`
   color: ${(props) => props.theme.colors.textSecondary};
 `;
 document.body.style = `background: ${theme.colors.background};`;
```

Comment:
This line of code is causing the color of the document outside of App to be set to a color, but isn't affected by the dark-mode changes.  Could you move this to line 47 and have it refer to themeToUse instead of theme?  Thanks!